src/main/java/uk/gov/companieshouse/items/orders/api/model/CertificateItemOptions.java

```diff
@@ -0,0 +1,282 @@
+        return isCertExtraI;
+    }
+
+    public void setCertExtraI(boolean certExtraI) {
+        isCertExtraI = certExtraI;
+    }
+
+    @JsonProperty("cert_inc")
+    public boolean isCertInc() {
+        return isCertInc;
+    }
+
+    public void setCertInc(boolean certInc) {
+        isCertInc = certInc;
+    }
+
+    @JsonProperty("cert_inc_con")
+    public boolean isCertIncCon() {
+        return isCertIncCon;
+    }
+
+    public void setCertIncCon(boolean certIncCon) {
+        isCertIncCon = certIncCon;
+    }
+
+    @JsonProperty("cert_inc_con_last")
+    public boolean isCertIncConLast() {
+        return isCertIncConLast;
+    }
+
+    public void setCertIncConLast(boolean certIncConLast) {
+        isCertIncConLast = certIncConLast;
+    }
+
+    @JsonProperty("cert_isscap")
+    public boolean isCertIssCap() {
+        return isCertIssCap;
+    }
+
+    public void setCertIssCap(boolean certIssCap) {
+        isCertIssCap = certIssCap;
+    }
+
+    @JsonProperty("cert_mem")
+    public boolean isCertMem() {
+        return isCertMem;
+    }
+
+    public void setCertMem(boolean certMem) {
+        isCertMem = certMem;
+    }
+
+    @JsonProperty("cert_mortdoc")
+    public boolean isCertMortDoc() {
+        return isCertMortDoc;
+    }
+
+    public void setCertMortDoc(boolean certMortDoc) {
+        isCertMortDoc = certMortDoc;
+    }
+
+    @JsonProperty("cert_nomc")
+    public boolean isCertNomc() {
+        return isCertNomc;
+    }
+
+    public void setCertNomc(boolean certNomc) {
+        isCertNomc = certNomc;
+    }
+
+    @JsonProperty("cert_nonexis")
+    public boolean isCertNonExis() {
+        return isCertNonExis;
+    }
+
+    public void setCertNonExis(boolean certNonExis) {
+        isCertNonExis = certNonExis;
+    }
+
+    @JsonProperty("cert_only")
+    public boolean isCertOnly() {
+        return isCertOnly;
+    }
+
+    public void setCertOnly(boolean certOnly) {
+        isCertOnly = certOnly;
+    }
+
+    @JsonProperty("cert_other_c")
+    public boolean isCertOtherC() {
+        return isCertOtherC;
+    }
+
+    public void setCertOtherC(boolean certOtherC) {
+        isCertOtherC = certOtherC;
+    }
+
+    @JsonProperty("cert_ret")
+    public boolean isCertRet() {
+        return isCertRet;
+    }
+
+    public void setCertRet(boolean certRet) {
+        isCertRet = certRet;
+    }
+
+    @JsonProperty("cert_roc")
+    public boolean isCertRoc() {
+        return isCertRoc;
+    }
+
+    public void setCertRoc(boolean certRoc) {
+        isCertRoc = certRoc;
+    }
+
+    @JsonProperty("cert_sec")
+    public boolean isCertSec() {
+        return isCertSec;
+    }
+
+    public void setCertSec(boolean certSec) {
+        isCertSec = certSec;
+    }
+
+    @JsonProperty("cert_shar")
+    public boolean isCertShar() {
+        return isCertShar;
+    }
+
+    public void setCertShar(boolean certShar) {
+        isCertShar = certShar;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
Can use Gson library for toString instead;

` @Override
    public String toString() {
        return new Gson().toJson(this);
    }`